packages/neon/neon/lib/src/utils/app_implementation.dart

```diff
@@ -53,6 +53,25 @@ abstract class AppImplementation<T extends Bloc, R extends NextcloudAppSpecificO
     );
   }
 
+  /// Route for the app.
+  ///
+  /// All pages of the app must be specified as subroutes.
+  /// If this is not [GoRoute] an inital route name must be specified by overriding [initialRouteName].
+  RouteBase get route => throw UnimplementedError();
```

Comment:
How about making this abstract? Then every app needs to explicitly set this. I think that leads to less confusion

Reply:
I'ts already on the todo. I made it this way so every single commit is valid and can compile